File: Uno/EjercicioUno.py
nombre = str(input("Ingrese su nombre: "))
salario = int(input("Ingrese salario por hora: "))
horas = int(input("Ingrese cantidad de horas: "))
trabajo = int(salario*horas)

if trabajo >= 450000:
  print("Su nombre es ", nombre , "su salario es: ", trabajo)
else:
  print("su nombre es: " , nombre )

File: Uno/EjercicioUno.cpp
#include <iostream>
using namespace std;
int main(){
string nombre;
int salario;
int horas;
  cout << "Ingrese nombre empleado: " << endl;
  cin >> nombre;
  cout << "Ingrese salario por hora: " << endl;
  cin >> salario;
  cout << "Ingrese horas trabajadas: " << endl;
  cin >> horas;

  int trabajo = salario * horas;

  if (trabajo >= 450000){
    cout <<"Su nombre es " <<nombre << " y su salario es: "<<trabajo<<endl;
  }else{
    cout <<"Su nombre es: "<<nombre << endl;
  } 
}